zigzag/xml_parsing_facade.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+# ======================================================================================================================
+# Imports
+# ======================================================================================================================
+
+from __future__ import absolute_import
+import os
+import pytest_rpc
+from lxml import etree
+from zigzag.test_log import TestLog
+
+
+class XmlParsingFacade(object):
+
+    _MAX_FILE_SIZE = 52428800
```

Comment:
What is the significance of this number?

Reply:
Its the max file size allowed by the qTest API.  
https://support.qasymphony.com/hc/en-us/articles/115002958146-qTest-API-Specification